core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -746,6 +746,16 @@ trait DeltaSQLConfBase {
       .internal()
       .booleanConf
       .createWithDefault(true)
+
+  val FAST_INTERLEAVE_BITS_ENABLED =
+    buildConf("fast.interleaveBits.enabled")
```

Comment:
i dont like this name. this interleaving is not really a feature by itself, but part of the larger zorder feature. so i think the name of conf should use the zorder conf naming convention `optimize.zorder.fastInterleaveBits.enabled`

also please colocate this conf with the other zorder confs

Reply:
SGTM, updated.